lua/luasnip/init.lua

```diff
@@ -270,7 +270,8 @@ local function snip_expand(snippet, opts)
 	local snip_parent_node = snip:trigger_expand(
 		session.current_nodes[vim.api.nvim_get_current_buf()],
 		pos_id,
-		env
+		env,
+		opts.skip_indent_nodes
```

Comment:
I think we should frame this the other way around, so instead of "skip indenting" with default false, we use just "indent" with default true.